include/rcutils/time.h

```diff
@@ -39,6 +39,8 @@ extern "C"
 #define RCUTILS_NS_TO_MS(nanoseconds) (nanoseconds / (1000LL * 1000LL))
 /// Convenience macro to convert nanoseconds to microseconds.
 #define RCUTILS_NS_TO_US(nanoseconds) (nanoseconds / 1000LL)
+/// Convenience macro for rcutils_steady_time_now(rcutils_time_point_value_t *).
+#define RCUTILS_STEADY_TIME rcutils_steady_time_now
```

Comment:
Is this being used?

Reply:
Yes, it's used by the [tests](https://github.com/ros2/rcutils/blob/3daf12617c2bfe7d6eb49c68e2710d7727ef2e6d/test/test_logging_macros.cpp#L157).
It was there by default, before these changes.

Reply:
I don't understand, how was it being set before, and why are you setting it here now? I only see this symbol (`RCUTILS_STEADY_TIME`) in one place in this pull request. Shouldn't remove it somewhere else or use it somewhere else for that to make sense?

Reply:
@wjwwood as @BMarchi mentioned earlier, it is used by tests:
https://github.com/ros2/rcutils/blob/3daf12617c2bfe7d6eb49c68e2710d7727ef2e6d/test/test_logging_macros.cpp#L156-L157

Up until this patch, `RCUTILS_STEADY_TIME` was undefined. That wasn't a problem because the throttled logging macro implementation ignored the corresponding macro argument (`time_source_type`) completely:

https://github.com/ros2/rcutils/blob/3daf12617c2bfe7d6eb49c68e2710d7727ef2e6d/resource/logging_macros.h.em#L176-L197

@BMarchi made that define expand to the right function to provide a nice API and keep those tests passing.

Reply:
Oh, I had no idea this worked...

```
#define FOO(arg1, arg2, arg3) \
  arg2 = arg3;

int main()
{
  int a, b = 1;
  FOO(DOES_NOT_EXIST, a, b)
  return 0;
}
```

That's awful, haha.

----

So then why is there no system time equivalent?

Also, why do you need a macro here, why couldn't you just pass `rcutils_steady_time_now`?

Reply:
@hidmic suggested on keep using the [macro](https://github.com/ros2/rcutils/pull/183#discussion_r330247026), so I just left it there

Reply:
That's fine, though I still wonder if a system time version makes sense.

I don't think it should block this.